internal/policy/container/runs_as_nonroot.go

```diff
@@ -33,20 +33,22 @@ func (p *RunAsNonRootCheck) getDataToValidate(image cranev1.Image) (string, erro
 	return configFile.Config.User, nil
 }
 
-func (p *RunAsNonRootCheck) validate(user string) (bool, error) {
+func (p *RunAsNonRootCheck) validate(ctx context.Context, user string) (bool, error) {
+	logger := logr.FromContextOrDiscard(ctx)
+
 	if user == "" {
-		log.L().Info("detected empty USER. Presumed to be running as root")
-		log.L().Info("USER value must be provided and be a non-root value for this check to pass")
+		logger.Info("detected empty USER. Presumed to be running as root")
+		logger.Info("USER value must be provided and be a non-root value for this check to pass")
 		return false, nil
 	}
 
 	if user == "0" || user == "root" {
-		log.L().Infof("detected USER specified as root: %s", user)
-		log.L().Info("USER other than root is required for this check to pass")
+		logger.Info(fmt.Sprintf("detected USER specified as root: %s", user))
```

Comment:
Maybe we don't need to Infof this? Seems like the log message tells us exactly what... user we found.

Reply:
Could make it say specified as root or UID 0... Agreed.